src/app/shared/constants.ts

```diff
@@ -33,7 +33,7 @@ export const servicesURLSegment = 'services';
 export const toolsURLSegment = 'tools';
 export const altAvatarImg = 'http://www.imcslc.ca/imc/includes/themes/imc/images/layout/img_placeholder_avatar.jpg';
 export const devMode = false;
-export const currentTOSVersion: User.TosversionEnum = User.TosversionEnum.TOSVERSION1;
```

Comment:
A little confused but cannot comment on it directly. 
`src/assets/docs/Dockstore_Terms_of_Service.pdf` shows as a deletion but I don't see an addition.

Reply:
You can view the file by clicking the three dots `...` on the right and hitting `view file`. That will open up the new TOS pdf.

Not sure why we can't view it normally though...

Reply:
Oh, never mind. Its (just) 81% smaller for some reason, its not fully deleted.